### PR TITLE
Added new utxo endpoints, including the original transaction

### PR DESCRIPTION
**Pull request by icellan**
This is still work in progress, needs bux to be updated first

### CODE REVIEW

File: actions/utxos/count.go [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import (
+	"net/http"
+
+	"github.com/BuxOrg/bux"
+	"github.com/BuxOrg/bux-server/actions"
+	"github.com/julienschmidt/httprouter"
+	apirouter "github.com/mrz1836/go-api-router"
+)
+
+// get will fetch utxos according to conditions
```
Comment (mrz1836) [inline actions/utxos/count.go:12]: Wrong comment

File: config/envs/development.json [hunk 2]
```diff
@@ -79,13 +80,14 @@
     "host": "localhost",
     "name": "xapi",
     "password": "",
-    "_port": "3306",
-    "port": "5432",
+    "port": "3306",
+    "_port": "5432",
```
Comment (mrz1836) [inline config/envs/development.json:84]: Use and env var vs hacking with _

File: config/envs/development.json [hunk 3]
```diff
@@ -79,13 +80,14 @@
...
     "replica": false,
     "skip_initialize_with_version": true,
     "time_zone": "UTC",
     "tx_timeout": "10s",
-    "user": "postgres"
+    "user": "root",
+    "_user": "postgres"
```
Comment (mrz1836) [inline config/envs/development.json:90]: Use and env var vs hacking with _

**Review by mrz1836 — CHANGES_REQUESTED** [review]
See my comments

File: config/envs/development.json [hunk 4]
```diff
@@ -32,15 +33,15 @@
     "uri": "mongodb://localhost:27017/xapi"
   },
   "monitor": {
-    "auth_token": "",
-    "bux_agent_url": "ws://localhost:8000/websocket",
+    "auth_token": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJzaWdnaSIsImV4cCI6MTc1MDM4MTA2OH0.EXHm4MrwnJGjHIrdSUP39TyQZsieukWivCwqecQO0F8",
```
[the remark this answers is not in the document]
Comment (icellan) [reply config/envs/development.json:36]: WTF?

**Comment by icellan** [comment]
Squashed the branch with the fixes from the comments you made

**Review by mrz1836 — APPROVED** [review]
LGTM

**Pull request merged**